cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/ProjectEndpointInput.java

```diff
@@ -106,6 +140,38 @@ public ProjectEndpointInput.GetAll pageSize(int pageSize) {
 
     }
 
+    public static final class ProjectChildLocations extends EndpointInput {
```

Comment:
How about calling this something like GetChildLocations? To prevent confusion with the ProjectChildLocations in the model.

Reply:
Good idea, updated